frontend/shared/components/src/groups/hooks/useFinancialSupportSettings.ts

```diff
@@ -1,21 +1,31 @@
 import { FinancialSupportSettings, Group, OrganizationRecordsConfiguration } from "@stamhoofd/structures";
 import { computed, Ref, unref } from "vue";
+import { useAppContext } from "../../context";
 import { useOrganization, usePlatform } from "../../hooks";
 
 export function useFinancialSupportSettings(options?: {group?: Ref<Group|null>|Group|null}) {
     const platform = usePlatform()
     const organization = useOrganization()
+    const group = unref(options?.group);
 
     const financialSupportSettings = computed(() => platform.value.config.financialSupport ?? FinancialSupportSettings.create({}) )
     const recordsConfiguration = computed(() => OrganizationRecordsConfiguration.build({
         platform: platform.value,
         organization: organization.value,
-        group: unref(options?.group),
+        group,
         includeGroup: true
     }))
 
+    let enabled = computed(() => recordsConfiguration.value.financialSupport);
+
+    if(!group) {
+        if(useAppContext() === 'admin') {
+            enabled = computed(() => true)
+        }
+    }
```

Comment:
Dit moet dus ook in de computed komen, in principe kan de buitenstaander de group veranderen, en dan moet het resultaat ook veranderen